src/logion/services/blockconsumption.service.ts

```diff
@@ -56,13 +59,25 @@ export class BlockConsumer {
                 updatedOn: now
             });
         }
-        this.syncPointRepository.save(lastSyncPoint);
+        await this.syncPointRepository.save(lastSyncPoint);
     }
 
     private async processBlock(blockNumber: bigint): Promise<void> {
-            const block = await this.blockService.getBlockExtrinsics(blockNumber);
-            await this.transactionSynchronizer.addTransactions(block);
-            await this.locSynchronizer.updateLocRequests(block);
-            await this.protectionSynchronizer.updateProtectionRequests(block);
+        const block = await this.blockService.getBlockExtrinsics(blockNumber);
+        const timestamp = this.extrinsicDataExtractor.getBlockTimestamp(block);
+        if (timestamp === undefined) {
+            throw Error("Block has no timestamp");
+        }
+        await this.transactionSynchronizer.addTransactions(block);
+        for (let i = 0; i < block.extrinsics.length; ++i) {
```

Comment:
Shouldn't it be the responsibility of each synchronizer to decide if it ignores failed extrinsics or not?

Reply:
> Shouldn't it be the responsibility of each synchronizer to decide if it ignores failed extrinsics or not?

Yes,good suggestion. Note that previously, all failed extrinsics were filtered out in method `successfulExtrinsics()`, now renamed to `allExtrinsics()`.

Reply:
Done.